addons/mail/static/src/new/web/messaging_menu/notification_item.js

```diff
@@ -26,4 +29,12 @@ export class NotificationItem extends Component {
     onClick(ev) {
         this.props.onClick(ev.target === this.markAsReadRef.el);
     }
+
+    get hasLeftSwipe() {
+        return this.props.onSwipeLeft && this.env.isSmall;
```

Comment:
Why is env.isSmall specifically checked only for notification item and not for other swipers?
It was previously checked for all, the current lack of consistency is strange.